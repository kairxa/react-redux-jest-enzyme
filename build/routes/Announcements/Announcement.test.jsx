/* eslint-disable import/no-extraneous-dependencies */
/* eslint-disable no-undef */

import React from 'react';
import renderer from 'react-test-renderer';

import Announcement from './Announcement';

test('should render equal to snapshot', () => {
  const owner = {
    name: 'Putra Prawira Tanzil',
    id: '123456789abcdef',
    picture: 'http://placehold.it/320x320',
  };
  const content = {
    title: '# This is a content\n\nLorem ipsum dolor sit amet lalala this is just a text for testing.\n\nOkay thanks.',
    cover: 'http://placehold.it/1600x900',
    createdAt: '2016-11-27T14:16:00.000Z',
  };
  const container = renderer.create(
    <Announcement
      owner={owner}
      content={content}
    />
  );
  const tree = container.toJSON();
  expect(tree).toMatchSnapshot();
});
