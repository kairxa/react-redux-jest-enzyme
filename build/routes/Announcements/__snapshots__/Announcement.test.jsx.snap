exports[`test should render equal to snapshot 1`] = `
<section
  className="containerAnnouncement">
  <header
    className="announcementHeader">
    <div
      className="profilePicture"
      style={
        Object {
          "backgroundImage": "url(http://placehold.it/320x320)",
        }
      } />
    <div
      className="profileName">
      <a
        className="profileLink"
        onClick={[Function]}
        style={Object {}}>
        Putra Prawira Tanzil
      </a>
      <span
        className="profileTime">
        Sun, Nov 27, 2016, 9:16 PM
      </span>
    </div>
    <div
      className="settings">
      <test-file
        className="icon" />
    </div>
  </header>
  <section
    className="announcementContent">
    <div>
      <h1
        data-sourcepos={undefined}>
        This is a content
      </h1>
      <p>
        Lorem ipsum dolor sit amet lalala this is just a text for testing.
      </p>
      <p>
        Okay thanks.
      </p>
    </div>
    <div
      className="announcementBanner">
      <img
        alt="Content"
        className="bannerImage"
        src="http://placehold.it/1600x900" />
    </div>
  </section>
</section>
`;
